### ISSUE TITLE
Parsing error with typecast() hint

### ISSUE THREAD

**Issue by nvanwyen** [nvanwyen/hive-odci#8]
If a typecast() parameter ends in a newline and the next parameter starts at the beginning of the next line then the parsing fails, because "spaces" only are used to delineate parameters.

Example
```
select /*+ typecast(
col1:number
col2:date
*/
* from my_table
/
```
Currently the parser will see this as ```typecast(col1:numbercol2:date)``` and an "Invalid parameter exception is thrown. The parser needs to reciginze this formatting option so that it is parsed as ```typecast(col1:number col2:date)``` instead.

**Comment by nvanwyen** [comment]
POC, duplicates issue ...

```
select *
from table( hive_q( 'select
/*+
typecast ( cust_id:number
last_name:varchar2(50)
first_name:varchar2(50)
username:clob
)
*/
cust_id
from cust
limit 10', null, null ) );
```

    without fix ...
```
        select *
        *
        ERROR at line 1:
        ORA-22369: invalid parameter encountered in method
        ORA-22369: invalid parameter encountered in method SetTypeInfo
        ORA-06512: at "HIVE.HIVE_T", line 42
        ORA-06512: at line 4
```

    with fix ...
```
           cust_id
        ----------
           1161115
           1168350
           1161269
           1161657
           1161937
           1162618
           1162875
           1163016
           1163934
           1164015

        10 rows selected.
```

Fix confirmed

**Comment by nvanwyen** [comment]
Fixed in release v0.1.5.55
